server.js

```diff
@@ -8,6 +8,11 @@ const PORT = 8080;
 const PUBLIC_FOLDER = __dirname + "/export/web"
 
 const server = http.createServer((request, response) => {
+    if (path.normalize(decodeURI(uri)) !== decodeURI(uri)) {
```

Comment:
uri does not exist at that moment

Reply:
Oh, you are right. But that's weird as well, we might have a bug when applying the patch. I'll take a look at it. Thanks for mentioning

Reply:
so you are spamming hundreds of repos with broken code?

Reply:
@phit man why would you consider this spam? We verified the vulnerability with the pipeline and the patch, but there is a bug with the patch verification method. Not all the pull requests ended up with this bug!
That's also another reason we are not sending all the pull requests at once. the actual number is over a thousand, I'm just trying to do it as good as possible but obviously, it takes time to make it better. 
We got mostly positive feedback until now but why do some people only see the negative part?
Maybe also check other reports as well instead of only checking the ones that are not merged?